docs/emails.rst

```diff
@@ -301,6 +301,12 @@ E-mail body can be changed in the database therefore reverse operation to ``sync
 
 Synchronize e-mail message status from the provider.
 
+
+``migrate_email_contents_to_files``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Migrates e-mail message contents from database to files. Required to be run when updating from version 0.5.0 or below.
```

Comment:
- When this PR is merged, please make major version 0.5.0
- The next version 0.6.0 will remove the `old_content` field. Before removing, it will check that the field is empty for all e-mail messages to avoid data loss. If it's not, it will instruct user to update to 0.5.0 first, run this command, and then proceed updating to 0.6.0